pages/view/_iscnId.vue

```diff
@@ -657,6 +661,14 @@ export default class ViewIscnIdPage extends Vue {
     return '';
   }
 
+  get isValidUrl() {
+    return (
+      this.metadata.usageInfo.startsWith('http://') ||
+      this.metadata.usageInfo.startsWith('https://') ||
+      this.metadata.usageInfo.startsWith('ftp://')
```

Comment:
Should check if `this.metadata` and `this.metadata.usageInfo` exists
i.e
```
      this.metadata?.usageInfo && (
      this.metadata.usageInfo.startsWith('http://') ||
      this.metadata.usageInfo.startsWith('https://') ||
      this.metadata.usageInfo.startsWith('ftp://'))
```
Not sure if this is valid syntax, please try